erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -55,7 +56,25 @@ def get_average_age(fifo_queue, to_date):
 
 	return flt(age_qty / total_qty, 2) if total_qty else 0.0
 
+def get_range_age(filters, fifo_queue, to_date):
+	range1 = range2 = range3 = above_range3 = 0.0
+	for item in fifo_queue:
```

Comment:
@AfshanKhan This is still not correct. The ranges should be added instead of just assigning the latest item count